### ISSUE TITLE
Plugin failed to update

### ISSUE THREAD

**Issue by waiotahi** [MerginMaps/qgis-plugin#471]
After installing QGIS 3.28 the Mergin plugin throws an error when updating
WARNING    Python error : Couldn't load plugin 'Mergin' due to an error when calling its classFactory() method See message log (Python Error) for more details.

**Comment by waiotahi** [comment]
No I don't use Dark theme / night mapping. However, after reloading QGIS again this morning I tried reinstalling the plugin and it installed with no errors - looks like its working fine

**Comment by NewGraphEnvironment** [comment]
since this issue deals with the plugin I will put this issue here.  maybe it should go elsewhere though...

On update to plugin yesterday I can't seem to get a sync off.  Is it looking for a svg that is not there?

Default theme

here is the error on attempted sync:

```
2023-10-12T10:18:09     WARNING    Traceback (most recent call last):
              File "/Users/airvine/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Mergin/plugin.py", line 716, in sync_project
              self.project_manager.project_status(self.path)
              File "/Users/airvine/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Mergin/projects_manager.py", line 170, in project_status
              dlg = ProjectStatusDialog(
              File "/Users/airvine/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Mergin/project_status_dialog.py", line 84, in __init__
              self.validate_project()
              File "/Users/airvine/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Mergin/project_status_dialog.py", line 226, in validate_project
              results = validator.run_checks()
              File "/Users/airvine/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Mergin/validation.py", line 107, in run_checks
              self.check_svgs_embedded()
              File "/Users/airvine/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Mergin/validation.py", line 344, in check_svgs_embedded
              symbols = renderer.symbols(context)
             AttributeError: 'NoneType' object has no attribute 'symbols'
```

**Comment by wonder-sk** [comment]
@NewGraphEnvironment thanks for reporting, this was a bug that got into 2023.4 release, but we fixed it quickly afterwards in 2023.4.1.

**Comment by alexbruy** [comment]
Seems all issues mentioned here are already fixed. Closing.
